account_banking_nl_ing_mt940/account_banking_nl_ing_mt940.py

```diff
@@ -20,36 +21,41 @@
 ##############################################################################
 import re
 from openerp.tools.translate import _
-from openerp.addons.account_banking.parsers.models import (
-    parser,
-    mem_bank_transaction,
-)
+from openerp.addons.account_banking.parsers.models import parser,\
+    mem_bank_transaction
 from openerp.addons.account_banking_mt940.mt940 import MT940, str2float
 
 
-class transaction(mem_bank_transaction):
+class Transaction(mem_bank_transaction):
+    """Override validity check in base class."""
     def is_valid(self):
-        '''allow transactions without remote account'''
+        """allow transactions without remote account"""
         return bool(self.execution_date) and bool(self.transferred_amount)
 
 
 class IngMT940Parser(MT940, parser):
+    """Define mt940 parser for Dutch ING bank."""
     name = _('ING MT940 (structured)')
     country_code = 'NL'
     code = 'INT_MT940_STRUC'
+    footer_regex = '^-}$'
```

Comment:
I used https://www.ing.nl/media/ING_ibp_mt940_942_format_description_the_netherlands_tcm162-45288.pdf as reference, there they state the message trailer is '-XXX' (p6) - did they release another document replacing this one?